tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -345,6 +345,7 @@ impl<T: Numeric> CudaVec<T> {
         }
     }
 
+    #[allow(clippy::needless_pass_by_ref_mut)]
```

Comment:
feels like a false positive ?

Reply:
I think it's because we use mut raw pointer which we don't need a &mut reference to access

Reply:
ok but it's better to have the &mut still here to enforce a single mutable access, thanks